app/javascript/components/applications/ApplicationPage.js

```diff
@@ -45,18 +44,21 @@ const buildActionLinkIfNecessary = (app, showAddBtn) => {
   return actions
 }
 
-const ApplicationPage = ({ isLeaseUp = false }) => {
+/**
+ * Non-lease up application page.
+ */
+const ApplicationPage = () => {
   const [application, setApplication] = useState(null)
   const [fileBaseUrl, setFileBaseUrl] = useState(null)
   const [loading, setLoading] = useState(true)
 
   const { applicationId } = useParams()
-  const [{ breadcrumbData }, actions] = useContext(AppContext)
+  const [, dispatch] = useAppContext()
```

Comment:
Note that now instead of passing actions, we're just passing the dispatch function.

So before you could call `action.doSomething()`, now you need to call `doSomething(dispatch)` and pass it the dispatch. The nice thing about this is you can jump to definition, and these are just regular functions (reduces the amount of magic)